bundles/org.openhab.io.imperihome/README.md

```diff
@@ -496,6 +496,14 @@ Switch MyShutterStop "Stop shutter" ["iss:room:Test", "iss:type:DevSwitch"]  { c
 
 Implement a Rule listening for the ON command on the switch to handle the stop action yourself.
 
+## Tagging in OH3
```

Comment:
At all times we try not to mention the version of openHAB because this means the documentation becomes outdated when V4 gets released. Simply refer to it as 'openHAB' instead.